komand/server.py

```diff
@@ -98,8 +102,76 @@ def handler(prefix, name, test):
                 r.status_code = status_code
                 return r
 
+        @app.route("/add_worker", methods=["POST"])
+        def add_worker():
+            """
+            Adds a worker (another process)
+            :return: Json Response
+            """
+            r = {}
+
+            # https://docs.gunicorn.org/en/stable/signals.html#master-process
+            try:
+                self.logger.info("Adding a worker")
+                self.logger.info("Current process is: %s" % self.master_pid)
+                os.kill(self.master_pid, signal.SIGTTIN)
+            except Exception as e:
+                r.status_code = 500
+                r.error = e
+                return jsonify(r)
+
+            r["num_workers"] = self._number_of_workers()
+            return jsonify(r)
+
+        @app.route("/remove_worker", methods=["POST"])
+        def remove_worker():
+            """
+            Shuts down a worker (another process)
+            If there is only 1 worker, nothing happens
+
+            :return: Json Response
+            """
+
+            r = {}
+
+            # https://docs.gunicorn.org/en/stable/signals.html#master-process
+            try:
+                self.logger.info("Removing a worker")
+                self.logger.info("Current process is: %s" % self.master_pid)
+                os.kill(self.master_pid, signal.SIGTTOU)
+            except Exception as e:
+                r = {}
+                r.status_code = 500
+                r.error = e
+                return jsonify(r)
+
+            return jsonify(r)  # Flask or Gunicorn expect a return
+
+        @app.route("/number_of_workers", methods=["POST"])
```

Comment:
To keep it a bit more restful, would recommend:
```suggestion
        @app.route("/workers", methods=["GET"])
```